src/style.css

```diff
@@ -0,0 +1,127 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
+
+.cards {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  max-width: 200px;
+  padding: 32px 16px 16px;
+}
+
+.cards__img {
+  margin-bottom: 40px;
+  width: 160px;
+  height: 134px;
+
+}
+
+.cards__text {
+  margin-right: 0;
+  margin-bottom: 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+  width: 166px;
+  height: 36px;
+}
+
+.cards__code {
+  align-self: start;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+
+}
+
+.cards__reviewsTag {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+  margin-bottom: 24px;
+  width: 100%;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images\star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--4 .stars__star:nth-of-type(-n + 4) {
+  background-image: url(images\star-active.svg);
+}
```

Comment:
![image](https://user-images.githubusercontent.com/53319174/179745440-d912d3c9-341c-4226-be87-cd144d44a283.png)

Use the styles from the previous task. The number of active stars should vary. If you change the modifier to 2, then there should be 2 active stars, etc.
![image](https://user-images.githubusercontent.com/53319174/179745657-58bef74d-a13b-4068-bf52-4a24f2b4d4af.png)